chef/cookbooks/nova/templates/default/libvirtd.conf.erb

```diff
@@ -271,13 +270,13 @@ auth_tcp = "<%= @libvirtd_auth_tcp %>"
 #
 # XXX this isn't actually enforced yet, only the per-client
 # limit is used so far
-#max_requests = 20
+max_requests = <%= @libvirtd_max_clients %>
 
 # Limit on concurrent requests from a single client
 # connection. To avoid one client monopolizing the server
 # this should be a small fraction of the global max_requests
 # and max_workers parameter
-#max_client_requests = 5
+max_client_requests = <%= @libvirtd_prio_workers %>
```

Comment:
Interesting, we were using the wrong libvirtd.conf template then. I've updated it to the SLE12 version and explicitely enabled the setting so that it is in effect for SLE11 as well.